src/Utilities/System/Abort.cpp

```diff
@@ -1,26 +1,16 @@
 // Distributed under the MIT License.
 // See LICENSE.txt for details.
 
-/// \file
-/// Defines function Parallel::abort.
-
-#pragma once
+#include "Utilities/System/Abort.hpp"
 
 #include <charm++.h>
 #include <csignal>
 #include <cstdlib>
 #include <exception>
 #include <string>
```

Comment:
Do we need all of those includes?  Shouldn't need `csignal` or `cstdlib` I think.

Reply:
Should be safe to delete them, done